### ISSUE TITLE
Modèle XGBoost

### ISSUE THREAD

**Issue by Vktrp** [Vktrp/Fraud-Detection#40]
- Entraînement XGBoost
- Tuning learning_rate, max_depth, subsample
- Early stopping sur validation set
- AUC-ROC >0.90
- Temps entraînement <10min
